### PR TITLE
Exposed sds sync interval as configuration

### PR DESCRIPTION
**Pull request by shtripat**
tendrl-bug-id: Tendrl/ceph-integration#299
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by shtripat** [comment]
@Tendrl/tendrl-core plz review

File: tendrl/ceph_integration/sds_sync/__init__.py [hunk 1]
```diff
@@ -122,7 +122,9 @@ def _run(self):
                     )
 
         while not self._complete.is_set():
-            gevent.sleep(10)
+            gevent.sleep(
```
Comment (r0h4n) [inline tendrl/ceph_integration/sds_sync/__init__.py:125]: This is not the only place that should use `sync_interval`

All the out of band object .save() with ttl should use ttl value by adding 10-15 seconds to `sync_interval`

**Review by r0h4n — CHANGES_REQUESTED** [review]

File: tendrl/ceph_integration/sds_sync/__init__.py (hunk 1 above)
Comment (shtripat) [reply tendrl/ceph_integration/sds_sync/__init__.py:125]: Should I grep for ttl and whenever there is ttl being set for the entities, change it to `sync_interval + 15` ?

Comment (shtripat) [reply tendrl/ceph_integration/sds_sync/__init__.py:125]: Currently there are no ttl references at the moment for entities in ceph-integration. When required we can use as `sync_interval + 15`

**Review by r0h4n — APPROVED** [review]

**Pull request merged**